NPM-2.md

```diff
@@ -0,0 +1,132 @@
+
+### Summary
+
+This methodology outlines the actions a potential Carbon Removal Claim ("CRC")
+Supplier must take to register a Soil based CRC-generating Project on the Nori platform.
+Once a Project is registered on the platform, the designated "Responsible
+Person" (the "Supplier") is permitted to submit a claim to Nori,
+instructing the Nori platform administrator to list verified CRCs generated by
+the Registered Project for sale on the Nori platform for up to 10 years, subject
+to complying with NMP-2, Recurring Annual Reporting Requirements.  During Year
+10, the Supplier may re-apply for Project Registration for a subsequent 10-year
+term.  Provide a simplified explanation of the project/methodology being
+proposed.
+
+### Establishing Project/CRC Ownership & the "Responsible Person"
+
+A single individual, resident in the United States of America, shall be
+designated the "Responsible Person" for purposes of communicating Project
+details and listing verified CRCs for sale on the Nori platform.  The
+Responsible Person must be the owner of the land on which the project is
+executed, or an American resident to whom authority to represent the land
+owner(s) has been formally assigned.  Proof of title to the project-affected
+lands and/or any relevant assignments must be submitted to Nori as a
+pre-registration requirement.  By registering a Project on Nori, the designated
+Person Responsible, hereinafter referred to as the "Supplier", is obliged to
+notify Nori of any changes in land ownership and/or assignments of the right to
+act as CRC Supplier for the Registered Project in a timely manner.
+
+### Establishing the Project CO2-Equivalent Baseline
+
+As a pre-condition of Nori Project Registration, the Supplier shall establish the
+Project's Original projected 10-year CO2 Baseline by registering with and
+meeting both the land management practice and energy use reporting requirements
+of a Nori white-listed model or soil carbon stock and flux simulation tool.
+Suppliers are required to report both changes in land management practices and
+energy use, for purposes of establishing the Project baseline. Note that most
+simulation tools require Suppliers to report up to 15 years of [historical land
+management practices](NMP-???) to establish the projected 10-year project baseline.
+
+### White-Listing Requirements
+
+For a carbon stock and flux simulation tool to be
+white-listed by Nori, the owners/administrators of the tool must commit to
+the equivalent of the following data privacy standard: The simulation tool
+owner/operator/administrator will not use or share the land management and
+energy use data the Supplier provides, except for the purposes the information
+provider directly approves.  The land management and energy use data that is
+used to establish the Project 10-year Projected CO2 Baseline will not be shared
+with Nori or stored in the Nori blockchain.
```

Comment:
We plan to write up the categories of data that are required to be public or private or are not required at all. For example:

  * public essential

    * location of the fields

  * public nonessential

    * satelite imagery

  * private essential

    * fertilizers and nutrients

  * private nonessential

    * what the farmers like to do in their personal time

Reply:
Has been started in NMP-3